vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -63,4 +95,50 @@ public ExhibitionAboutPage getExhibitionAboutPage() {
         return aboutPageList != null && !aboutPageList.isEmpty() ? aboutPageList.get(0):new ExhibitionAboutPage();
     }
     
+    /**
+     * This method maps the title in ExhibitionAboutPage, and add that to
+     * exhibitionTitles list for each user selected Exhibition Language.
+    */
+    public void setAboutPageTitle(LanguageDescriptionObject title, ExhibitionAboutPage exhibitionAboutPage) {
+        if(title!=null) {
+
+            ExhibitionLanguage exhibitionLanguage = exhibitionLanguageRepository.findByLabel(title.getExhibitionLanguage().getLabel());
+            if(exhibitionLanguage != null) {
+                title.setExhibitionLanguage(exhibitionLanguage);
+            }
+            Optional<ILanguageDescriptionObject> text = exhibitionAboutPage.getExhibitionTitles().stream().filter(exhibitionTitle-> {
+                return  exhibitionTitle.getExhibitionLanguage().equals(exhibitionLanguage);
+            }).findAny();
+            if(text.isPresent()) {
+                text.get().setText(title.getText());
+            } else {
+                exhibitionAboutPage.getExhibitionTitles().add(title);
+            }
+        }
```

Comment:
I'm confused why all of this is necessary. The `LanguageDescriptionObject` have ids, don't they? why not pass along the ids and use those?